src/seaport.ts

```diff
@@ -278,27 +289,26 @@ export class OpenSeaPort {
    * Emits the `PurchaseAssets` event when the transaction is prompted.
    * @param param0 __namedParameters Object
    * @param numTokensToBuy The number of WrappedNFT tokens to purchase and unwrap
-   * @param amountInEth The estimated cost in wei for tokens (probably some ratio above the minimum amount to avoid the transaction failing due to frontrunning, minimum amount is found by calling UniswapExchange(uniswapAddress).getEthToTokenOutputPrice(numTokensToBuy.mul(10**18));
+   * @param amount The estimated cost in wei for tokens (probably some ratio above the minimum amount to avoid the transaction failing due to frontrunning, minimum amount is found by calling UniswapExchange(uniswapAddress).getEthToTokenOutputPrice(numTokensToBuy.mul(10**18));
    * @param contractAddress Address of the corresponding NFT core contract for these NFTs.
    * @param accountAddress Address of the user's wallet
    */
   public async purchaseAssets(
-      { numTokensToBuy, amountInEth, contractAddress, accountAddress }:
+      { numTokensToBuy, amount, contractAddress, accountAddress }:
       { numTokensToBuy: number;
-        amountInEth: number;
+        amount: BigNumber;
         contractAddress: string;
         accountAddress: string; }
     ) {
 
     const token = WyvernSchemas.tokens[this._networkName].canonicalWrappedEther
```

Comment:
this is unused now. i think amountInEth was better, that way it's easier to use. only the comment needed to change